.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+# See CONTRIBUTING.md for instructions.
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+# Commitizen runs in commit-msg stage
+# but we don't want to run the other hooks on commit messages
+default_stages: [commit]
+
+repos:
+  # Check formatting and lint for starlark code
+  - repo: https://github.com/keith/pre-commit-buildifier
+    rev: 4.0.1.1
+    hooks:
+      - id: buildifier
+      - id: buildifier-lint
+  # Enforce that commit messages allow for later changelog generation
+  - repo: https://github.com/commitizen-tools/commitizen
```

Comment:
I'm pretty sure that commitizen should be part of the setup, so that contributors remember to indicate fix/feature/breaking change. Otherwise it's so hard to determine next release version.

It will remind you about the `--no-verify` option whenever it fails, so when I'm doing quick hacking like that I just add the option.